### PR TITLE
scripts: xtensa-build-all: Add support for > 1 makefile job

### PR DESCRIPTION
**Pull request by lgirdwood**
Add -j option to allow setting Makefile jobs.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>

### CODE REVIEW

**Comment by jajanusz** [comment]
Just a little note:
You can just pass "-j" to make what means something like "-j<infinity>" (it's useful with xcc). However with this script you cannot do this. If you pass "-j" it will just build like -j1.

File: scripts/xtensa-build-all.sh [hunk 1]
```diff
@@ -12,6 +14,7 @@ then
 	echo "       [-l] Build rimage locally"
 	echo "       [-a] Build all platforms"
 	echo "       [-d] Enable debug build"
+	echo "       [-j n] Set number of make build jobs"
```
Comment (jajanusz) [inline scripts/xtensa-build-all.sh]:
```suggestion
	echo "       [-j [n]] Set number of make build jobs. Infinite jobs with no arg."
```

File: scripts/xtensa-build-all.sh [hunk 2]
```diff
@@ -28,18 +31,30 @@ else
 			then
 			BUILD_DEBUG=yes
 
+		elif [[ "$args" == "-j" ]]
+			then
+			BUILD_JOBS_NEXT=1
```
Comment (jajanusz) [inline scripts/xtensa-build-all.sh]:
```suggestion
			BUILD_JOBS_NEXT=1
			BUILD_JOBS=""
```

File: scripts/xtensa-build-all.sh [hunk 3]
```diff
@@ -28,18 +31,30 @@ else
...
 		elif [[ "$args" == "-a" ]]
 			then
 			PLATFORMS=${SUPPORTED_PLATFORMS[@]}
 		else
+			# check for plaform
 			for i in ${SUPPORTED_PLATFORMS[@]}
 			do
 				if [ $i == $args ]
 				then
 					PLATFORMS+=$i" "
```
Comment (jajanusz) [inline scripts/xtensa-build-all.sh]:
```suggestion
					PLATFORMS+=$i" "
					BUILD_JOBS_NEXT=0
```

**Comment by lgirdwood** [comment]
@jajanusz good idea for -j and new beta "commit" feature is nice too (although it would be better if it allowed squashing).

**Comment by jajanusz** [comment]
@lgirdwood Yea, I expected it to be 1 commit because it's 1 review, I hope they will improve it over time

**Pull request merged**